operate/meta-backup.mdx

```diff
@@ -59,7 +59,7 @@ Example output:
 Here's an example of how to delete a meta snapshot with `risectl`:
 
 ```sql
-risectl meta delete-meta-snapshots [snapshot_ids]
+risectl meta delete-meta-snapshots --snapshot-ids <snapshot_id> [--snapshot-ids <snapshot_id>...]
```

Comment:
Added `[--snapshot-ids <snapshot_id>...]` to explain that to provide multiple snapshot ids, we need to repeate the option. We cannot do `--snapshot-ids 123 456`. Feel free to edit as I am not sure about the standard syntax to describe this.